Runtimes/Supplemental/cmake/modules/FindSwiftCore.cmake

```diff
@@ -0,0 +1,111 @@
+#[=======================================================================[.rst:
+FindSwiftCore
+------------
+
+Find SwiftCore, deferring to SwiftCoreConfig.cmake when requested.
+This is meant to find the core library to be linked by the Supplemental libraries.
+
+Imported Targets
+^^^^^^^^^^^^^^^^
+
+The following :prop_tgt:`IMPORTED` TARGETS may be defined:
+
+ ``SwiftCore``
+
+Hint Variables
+^^^^^^^^^^^^^^
+
+ ``SDKROOT``
+   Set the path to the Swift SDK Root.
+   This only affects MacOS builds.
```

Comment:
This description seems stale now -- this seems to be now an environment variable sensed when building on Windows

Reply:
The `SDKROOT` env var gets set up by the Windows toolchain installation. It's less of a hint and more meant as a fallback to avoid requiring the variable be set when building for Windows. If it's important, it should likely be set via the `Swift_SDKROOT` variable directly.